lib/all-badge-examples.js

```diff
@@ -0,0 +1,2035 @@
+'use strict';
+
+const allBadgeExamples = [
+  {
+    sectionId: 'build',
+    sectionName: 'Build',
+    badges: [
+      {
+        title: 'Travis',
+        previewBadgeUri: '/travis/rust-lang/rust.svg',
+        exampleBadgeUri: 'https://img.shields.io/travis/USER/REPO.svg'
```

Comment:
Maybe omit the hostname from here? We can prepend it in the rendering code (which would also avoid showing prod URLs in dev environments)

I wonder if it'd be better to have a URL pattern to remove the repetition... Something like:
```
pattern: '/travis/{USER}/{REPO}.svg`
previewParams: {
  USER: 'rust-lang',
  REPO: 'rust',
}
```

Reply:
> Maybe omit the hostname from here?

This makes sense. Though we'll need to sort out more clearly what's happening with `sed` in the Makefile. It depends on the hostname being there.

```
uriFormat: '/travis/{USER}/{REPO}.svg`,
previewParams: {
  USER: 'rust-lang',
  REPO: 'rust',
}
```

That looks amazing! The Sinatra format could be nice too (i.e. '/travis/:user/:repo.svg`. It's going to take some care. Let's do that in the next pass.

Reply:
I started a checklist in the top comment to make sure I don't miss anything.